pkg/ds/daemon_set.go

```diff
@@ -360,14 +395,6 @@ func (ds *daemonSet) schedule(node types.NodeName) error {
 	if err != nil {
 		return util.Errorf("Error setting label: %v", err)
 	}
-
-	// Will set the following the value, ds.Manifest, to the following key:
-	// <kp.INTENT_TREE>/<node>/<ds.Manifest.ID()>
-	// eg intent/127.0.0.1/test = test_pod
-	_, err = ds.kpStore.SetPod(kp.INTENT_TREE, node, ds.Manifest)
```

Comment:
did you mean to remove this? now the function only sets a label and does not set the pod?